client/txn.go

```diff
@@ -46,6 +46,7 @@ type txnSender struct {
 func (ts *txnSender) Send(ctx context.Context, call Call) {
 	// Send call through wrapped sender.
 	call.Args.Header().Txn = &ts.txn
+	call.Args.Header().Timestamp = ts.opTimestamp
```

Comment:
This doesn't really make sense. You shouldn't be able to change the timestamp within a transaction. The transaction timestamp should take precedence.